x-pack/plugin/sql/qa/build.gradle

```diff
@@ -16,8 +16,12 @@ dependencies {
 
   // CLI testing dependencies
   compile project(path: xpackModule('sql:sql-cli'), configuration: 'nodeps')
-  compile "org.orbisgis:h2gis-ext:1.3.2"
- 
+  
+  // H2GIS testing dependencies
+  compile ("org.orbisgis:h2gis:1.5.0") {
```

Comment:
I opt for externalizing h2gis (and h2 while at it) versions into a central place (like `jlineVersion), namely `build.gradle` under `sql`: https://github.com/elastic/elasticsearch/blob/f36d6a96e4b38a2a25eacd4692870ef93e1d4a3e/x-pack/plugin/sql/build.gradle#L13

Reply:
:+1: I will do that in master first and then make corresponding changes in the branch after merging

Reply:
I opened #40551 to address this in master. I will fix it for h2gis when I merge master into geosql branch.